docs/blog/2018-07-20-why-we-built-the-site-showcase/index.md

```diff
@@ -17,7 +17,7 @@ The Site Showcase was developed by Cassie Beckley ([@ThatOtherPerson](https://gi
 
 ## Submitting your site
 
-If you've built a site with Gatsby and would like it to appear in the Site Showcase, please [submit your site](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/site-showcase-submissions.md).
+If you've built a site with Gatsby and would like it to appear in the Site Showcase, please [submit your site](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/site-showcase-submissions.md).
```

Comment:
We should link to page on gatsbyjs.org (not to github) if possible.
```suggestion
If you've built a site with Gatsby and would like it to appear in the Site Showcase, please [submit your site](/contributing/site-showcase-submissions/).
```

Reply:
Good point -- I just pushed a commit to link it to https://www.gatsbyjs.org/contributing/site-showcase-submissions/. Does that work?

Reply:
Yeah, thanks!